content/trial_getting_started/set_trial_manager/getting-started-with-cloudify-docker-and-cli.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Local Command Line Interface "
+description = "This guide illustrates how to use cloudify docker image as local CLI client"
+weight = 95
+alwaysopen = false
++++
+
+{{%children style="h2" description="true"%}}
+
+The Cloudify Docker image comes with [Cloudify CLI] ({{< relref "cli/" >}}) pre installed.
+This guide illustrates how to use cloudify docker image as local CLI client.
+
+## Prerequisites
+
+Install Cloudify on your local desktop.
```

Comment:
Why are you using the community version here? what if I wish to use the premium trial docker which is actually the version that we recommend?
I would remove this sentence completely and just mention that you should have a docker deployed.

Reply:
in the context of trying out the docker image as a CLI client we only need an image with the CLI pre-installed - whether the manager is a premium or community is less relevant. The community has the advantage that I can install and run it in the simplest way possible.